starboard/shared/pthread/thread_create.cc

```diff
@@ -54,7 +54,7 @@ void* ThreadFunc(void* context) {
 
   delete thread_params;
 
-#if !SB_HAS_QUIRK(THREAD_AFFINITY_UNSUPPORTED)
+#if defined(_GNU_SOURCE)  // sched_setaffinity is Linux-only
```

Comment:
"GNU extension" instead of "Linux-only"?
https://www.gnu.org/software/libc/manual/html_node/CPU-Affinity.html

Reply:
Updated, thanks